README.md

```diff
@@ -55,22 +74,29 @@ ansible-lint site.yml
 ## Setting up a new staging server
 
 1. Add the new server to the `hosts` file.
-2. Get the root user password
-3. Optionally add public key(s) to `files/admin-ssh-keys`
-4. Then:
+2. Optionally add public key(s) to `files/admin-ssh-keys`
+3. Run playbook: `ansible-playbook -i hosts -l staging2.ceresfairfood.org.au site.yml -u root -k` # root user password defaults to Rimu login
+4. `sudo apt-get install libmariadb-dev`
+5. Push codebase and install app: `git push staging`
 
+Load database, eg: 
 ```
-ansible-playbook -i hosts -l staging2.ceresfairfood.org.au site.yml -u root -k
 ssh fairfood@staging2.ceresfairfood.org.au -A
 $ ssh staging.ceresfairfood.org.au "mysqldump -u root fairfood_production | gzip" > old_server.sql.gz
 $ zcat old_server.sql.gz | mysql fairfood_production
+```
 
-# Update the domain, then:
+If DNS is not pointing to this server yet, update it and obtain new certificate:
+```
 ssh root@staging2.ceresfairfood.org.au
 $ # Add server name to nginx config.
 $ certbot -d staging2.ceresfairfood.org.au -d staging.ceresfairfood.org.au --expand --renew-hook 'systemctl reload nginx'
+```
 
-# Disable root login.
+Finally, **remember to disable root login.**
+```
+sudo vim /etc/ssh/sshd_config
+# PermitRootLogin no
```

Comment:
In ofn-install, we do that with a special role. It does some more secure ssh settings. I hadn't included it in Fairfood because I didn't know how to use Ansible without the root user or a general passwordless sudo user. But we know that now.